### PR TITLE
Add italian translation file

### PR DESCRIPTION
**Pull request by kroniapp**

### CODE REVIEW

File: src/picker/i18n/it.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  skinToneLabel: 'Scegli una tonalità della pelle (corrente {skinTone})',
+  skinTonesLabel: 'Tonalità della pelle',
+  skinTones: [
+    'Default',
+    'Light',
+    'Medium-Light',
+    'Medium',
+    'Medium-Dark',
+    'Dark'
+  ],
```
Comment (nolanlawson) [inline src/picker/i18n/it.js:21]: It looks like this wasn't translated? ^

**Comment by nolanlawson** [comment]
Thanks a lot! Just one small question. ^

**Comment by paulvitry** [comment]
Hey, thanks for your work ! When do you think this PR will be merge ?

**Comment by kroniapp** [comment]
Sorry for the delay, I haven't touched the project for a long time... I've just edited the file, I hope it's ok now

**Comment by nolanlawson** [comment]
Looks good, thank you! :heart:

**Review by nolanlawson — APPROVED** [review]

**Pull request merged**